Bottom: cbcd6f96a8e1837e911c3c4fff5efdf70eb05e11
Top:    d10ac357735036bfdd1ae489b820ae993a80a248
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-30 00:14:29 +0100

Refresh of hid-gerber-don-t-use-the-core-

---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index 286d4ac..baec5ff 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -532,10 +532,10 @@ gerber_expose (HID * hid, BoxType *drawn_area, void *item)
   CountHoles (&nplated, &nunplated, drawn_area);
 
   if (nplated && set_layer ("plated-drill", SL (PDRILL, 0)))
-    DrawHoles (true, drawn_area);
+    DrawHoles (true, false, drawn_area);
 
   if (nunplated && set_layer ("unplated-drill", SL (UDRILL, 0)))
-    DrawHoles (false, drawn_area);
+    DrawHoles (false, true, drawn_area);
 
   if (set_layer ("componentmask", SL (MASK, TOP)))
     DrawMask (COMPONENT_LAYER, drawn_area);
